sections/image-banner.liquid

```diff
@@ -57,12 +76,12 @@
       >
     </div>
   {%- endif -%}
-  <div class="banner__content page-width" style="align-items: {{ section.settings.desktop_text_box_position }};">
```

Comment:
Oh, okay. Just remembered why I made this change.

It's because it's now applied on Desktop only, under a `@media screen and (min-width: 750px)` query. On mobile, we'll always use `align-items: center`.

Reply:
That makes sense 🙏 thanks for the explanation